docs/user_api/zfit.pdf.rst

```diff
@@ -78,3 +78,15 @@ They offer more or less support and freedom.
     :maxdepth: 2
 
     pdf/custom_base
+
+Physics PDFs
```

Comment:
@ikrommyd FYI, these are the parts of adding docs: I've added here a toctree that references a file above, the physics.rst situated in the folder "pdf", which is also created.